scripts/install

```diff
@@ -75,7 +80,21 @@ ynh_app_setting_set $app port $port
 # INSTALL DEPENDENCIES
 #=================================================
 
-ynh_install_app_dependencies postgresql-9.4 yarn ffmpeg
+# Import debian archive pubkey, need on ARM arch
+# taken from https://github.com/YunoHost-Apps/mastodon_ynh/blob/7db022c2cc3d6e9fa2e26803c4a424a3bb8c9bd4/scripts/install#L55
+arch=$(uname -m)
+if [[ $arch = arm*  ]]; then
```

Comment:
Je n'aime pas trop l'utilisation de dépôt tiers surout pour l'utilisation de logiciels présents déjà dans les dépôts officiels. Peux-tu confirmer le nécessité de l'utilisation de ce dépôt ?

Reply:
since I have no way to check on a RPi, I'll delete the lines. But since I took that from the [mastodon package](https://github.com/YunoHost-Apps/mastodon_ynh/blob/master/scripts/install#L55), they should have experienced the situation, right?

Reply:
I totally agree with frju365, adding a repository is a bad idea.
If you really need another version of a package than the one you can find in the usuals repo, it would be better to download the deb you need, than check the checksum.
Because, with a complete repository, the users can have some upgrade from this repo with all the problems comming with them.

So, even for a community app, please do not do that.